### ISSUE TITLE
UploadWizard-Konfiguration anpassen: Vorlage:Information ist englisch definiert.

### ISSUE THREAD

**Issue by dktue** [Tuepedia/Skinopt#36]
Erzeugt aus @abilustue Notiz

**Comment by dktue** [comment]
@abilustue : Ist das noch aktuell?

**Comment by abilustue** [comment]
Ja, das sorgt leider für nicht besonders gut gefüllte Bildbeschreibungen, wäre toll, du könntest das mal mit deinem Vorlagenknowhow prüfen. Ich hänge später mal einen Screenshot mit den Configs dazu in der localsettings.php an.

**Comment by abilustue** [comment]
Da gibt es eigentlich eine Art von Internationalisierungsfunktion, die Vorlage ist jedenfalls deutsch, die Daten, die der Upload-Wizard erhebt englisch. 
![image](https://user-images.githubusercontent.com/43342934/47217627-3f50e500-d3a9-11e8-975a-b3554446dd7f.png)

**Comment by abilustue** [comment]
Beispieldatei: https://www.tuepedia.de/wiki/Datei:Stockach_Altes_Feuerwehrhaus.jpg

**Comment by abilustue** [comment]
So sieht der Installationscode der extension in der localsettings.php aus:
![grafik](https://user-images.githubusercontent.com/43342934/47609501-159a5c80-da40-11e8-8d40-a2fa73b9d124.png)
language (de) und Hinweistext habe ich angepasst.
